### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by tisnik**
This is part for https://github.com/RedHatInsights/insights-operator-controller/issues/107

### CODE REVIEW

**Comment by tisnik** [comment]
Yup @lina-nikiforova it would be a nice enhancement. Do you happen to have an example how it should look like please?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
Comment (lina-nikiforova) [inline PULL_REQUEST_TEMPLATE.md]: I don't have an example but maybe something like this

```suggestion
- [ ] This change requires a documentation update

## Testing steps
Please describe how the change was tested locally. If, for some reason, the testing was not done or not done fully, please describe what are the testing steps.
```

Comment (tisnik) [reply PULL_REQUEST_TEMPLATE.md]: ok good!

**Comment by tisnik** [comment]
thank YOU @lina-nikiforova

**Pull request merged**